### ISSUE TITLE
Any hope of using Lu Wang's vim.js?

### ISSUE THREAD

**Issue by rockpunk** [sn-extensions/vim-editor#6]
Lu's js clone (https://github.com/coolwanglu/vim.js) seems to be a more user-friendly and intuitive experience compared with codemirror. Lu's clone also handles a faster key-repeat/low latency responsiveness. 

Also, the js dialog for command entry that's currently in the plugin is pretty suboptimal (not sure if this is a standard notes or a codemirror limitation, though.) 

Anyway, just figured I'd throw this out there to see if there's any traction. Thanks!

**Comment by atmoio** [comment]
Hmm, indeed, looks very nice. But looks like it's been abandoned? Also not clear from a quick glance whether it's fully client-side, or whether it requires some server component..

**Comment by rockpunk** [comment]
Yeah, doesn't look actively developed, but it's basically a cherry-picked version of the original vim c code compiled down to JS using emscripten (https://emscripten.org) <-- which is pretty frickin awesome, actually.

**Comment by atmoio** [comment]
Very interesting, will have to dig into that!

**Comment by rockpunk** [comment]
just for fun, i downloaded vim.js and played around with building it w/ emscripten. The compilation is a bit of pain, but seems to work on mac:

```
$ git clone https://github.com/emscripten-core/emsdk
$ 
```

**Comment by rockpunk** [comment]
sorry, accidentally closed this with a half-created comment. re-open and deleted comment =)
